--- Zend/Application/Module/Bootstrap.php	(revision 14208)
+++ Zend/Application/Module/Bootstrap.php	(working copy)
@@ -69,8 +69,8 @@
                 $r = new ReflectionClass($this);
                 $path = $r->getFileName();
                 $this->setResourceLoader(new Zend_Application_Module_Autoloader(array(
-                    'prefix' => $prefix,
-                    'path'   => dirname($path),
+                    'namespace' => $prefix,
+                    'basePath'   => dirname($path),
                 )));
             }
         }
Index: Zend/Application/Resource/Modules.php
===================================================================
--- Zend/Application/Resource/Modules.php	(revision 14208)
+++ Zend/Application/Resource/Modules.php	(working copy)
@@ -23,7 +23,7 @@
 /**
  * @see Zend_Application_Bootstrap_Resource_Base
  */
-require_once 'Zend/Application/Bootstrap/Resource/Base.php';
+require_once 'Zend/Application/Resource/Base.php';
 
 /**
  * Module bootstrapping resource
@@ -51,14 +51,13 @@
         $modules = $front->getControllerDirectory();
         $default = $front->getDefaultModule();
         foreach (array_keys($modules) as $module) {
-            if ($module === $default) {
-                continue;
-            }
+//            if ($module === $default) {
+//                continue;
+//            }
 
             $path = $front->getModuleDirectory($module);
             $bootstrapPath  = $path . '/Bootstrap.php';
             $bootstrapClass = ucfirst($module) . '_Bootstrap';
-            
             if (file_exists($bootstrapPath)) {
                 include_once $bootstrapPath;
                 
@@ -66,7 +65,7 @@
                     throw new Zend_Application_Resource_Exception('Bootstrap file found for module "' . $module . '" but bootstrap class "' . $bootstrapClass . '" not found');
                 }
                 
-                $moduleBootstrap = new $bootstrapClass($this);
+                $moduleBootstrap = new $bootstrapClass($this->getBootstrap());
                 $moduleBootstrap->bootstrap();
             }
         }
